### PR TITLE
dev/reference/tools/dm: update the column-mapping syntax

### PR DESCRIPTION
**Pull request by yikeke**
Via: https://github.com/pingcap/docs/pull/1166/ PTAL @kennytm @lilin90

### CODE REVIEW

File: dev/reference/tools/data-migration/features/overview.md [hunk 1]
```diff
@@ -378,41 +378,54 @@ column-mappings:
...
+| ☑ 已定义   | ☐ 空       | ☐ 空      | [`S`: 1 bit] [`I`: 4 bits] [`P`: 59 bits] |
 
-int64 比特表示 `[1:1 bit] [2:4 bits] [3:7 bits] [4:8 bits] [5:44 bits]`
-- 1：符号位，保留
-- 2：instance ID，默认 4 bits
-- 3：schema ID，默认 7 bits
-- 4：table ID，默认 8 bits
-- 5：自增主键 ID，默认 44 bits
+- `S`：符号位，保留
+- `I`：instance ID，默认 4 比特位
```
Comment (kennytm) [inline dev/reference/tools/data-migration/features/overview.md]: Either use "bit" here or "比特位" above. Consistency.

Comment (yikeke) [reply dev/reference/tools/data-migration/features/overview.md]: I changed all "bit(s)" into "比特位". PTAL again.

File: dev/reference/tools/data-migration/features/overview.md [hunk 2]
```diff
@@ -378,41 +378,54 @@ column-mappings:
...
+`partition id` 目的是为了解决分库分表合并同步的自增主键的冲突。
 
 **`partition id` 限制**
 
 注意下面的限制：
 
 - 只支持类型为 bigint 的列，通常为自增主键，联合主键或者联合唯一索引的其中一列
-- 库名的组成必须为 `schema 前缀 + 数字（即 schema ID）`，例如：支持 `s_1`, 不支持 `s_a`
-- 表名的组成必须为 `table 前缀 + 数字（即 table ID）`
+- 如果 `schema 前缀`不为空，则库名的组成必须为 `schema 前缀`或者 `schema 前缀 + 分隔符 + 数字（即 schema ID）`，例如：支持 `s` 和 `s_1`，不支持 `s_a`
```
Comment (lilin90) [inline dev/reference/tools/data-migration/features/overview.md]:
```suggestion
- 如果 `schema 前缀` 不为空，则库名的组成必须为 `schema 前缀` 或者 `schema 前缀 + 分隔符 + 数字（即 schema ID）`，例如：支持 `s` 和 `s_1`，不支持 `s_a`
```

File: dev/reference/tools/data-migration/features/overview.md [hunk 3]
```diff
@@ -378,41 +378,54 @@ column-mappings:
...
+- 如果 `table 前缀`不为空，则表名的组成必须为 `table 前缀`或者 `table 前缀 + 分隔符 + 数字（即 table ID）`
```
Comment (lilin90) [inline dev/reference/tools/data-migration/features/overview.md]:
```suggestion
- 如果 `table 前缀` 不为空，则表名的组成必须为 `table 前缀` 或者 `table 前缀 + 分隔符 + 数字（即 table ID）`
```

File: dev/reference/tools/data-migration/features/overview.md [hunk 4]
```diff
@@ -378,41 +378,54 @@ column-mappings:
...
+- 如果库名/表名不包含 `… + 分隔符 + 数字`部分，则对应的 ID 默认为 0
```
Comment (lilin90) [inline dev/reference/tools/data-migration/features/overview.md]:
```suggestion
- 如果库名/表名不包含 `… + 分隔符 + 数字` 部分，则对应的 ID 默认为 0
```

**Comment by yikeke** [comment]
Updated. PTAL again. @lilin90

**Pull request merged**